src/plugins/explore/public/application/legacy/discover/application/components/visualizations/line/line_chart_rules.ts

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
should these not be in legacy? the legacy directory was created for the duplicated files, new components or refactored ones can be outside and we'll make legacy empty eventually

Reply:
moved out of legacy